sunpy/net/helioviewer.py

```diff
@@ -146,7 +130,7 @@ def download_jp2(self, date, directory=None, overwrite=False, **kwargs):
 
         return self._get_file(params, directory, overwrite=overwrite)
 
-    def download_png(self, date, image_scale, layers, directory=None,
+    def download_png(self, date, image_scale, layers, eventLabels=False, directory=None,
                      overwrite=False, **kwargs):
         """Downloads a PNG image using data from Helioviewer.org.
```

Comment:
So this is the `takeScreenshot` from the API? We should probably mention that somewhere. Ideally if you can make our functions to the official names and note if they are different in the documentation string.

Reply:
Sorry I didn't get that

Reply:
So `download_png` isn't a name of a function from the official API that I could search for. The closest I could find was `takeScreenshot`

It would be nice if in the documentation strings we mention what function from the API this enables/uses.